LEAF_Request_Portal/FormWorkflow.php

```diff
@@ -120,7 +120,7 @@ public function getCurrentSteps(): ?array
                         ':quadGroupIDs' => $quadGroupIDs
                     );
                     $strSQL = 'SELECT * FROM services
-                        WHERE groupID IN (:quadGroupIDs)
+                        WHERE find_in_set(groupID, :quadGroupIDs)
```

Comment:
Not seeing the need for this change and it adds cycles to the computation.  "Where/In" just returns a boolean.  "find_in_set" has to compute the location and add 1.  And you're not using the return except as a boolean.

Reply:
If there is a more efficient way to do this, I can update it, but groupID IN was not consistently getting query results when the user was in more than one group.

Reply:
Do you have an example of a trouble query you could share? (Probably not in this thread.)